resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.service.erb

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=<%= EZBake::Config[:project] %> Service
+After=syslog.target network.target
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/sysconfig/<%= EZBake::Config[:project] %>
```

Comment:
I belive in this case, yes, it is correct. We're not installing anything to /etc/default for debian in ezbake

puppet-agent also has this same directory structure https://github.com/puppetlabs/puppet/blob/master/ext%2Fsystemd%2Fpuppet.service#L7